tests/test_requests.py

```diff
@@ -344,6 +344,33 @@ def test_display_leaderboard_for_http_error_404(self):
         assert response == expected
 
 
+class TestMakeSubmissionWhenSubmittingToChallengesUserDoesntParticipate(BaseTestClass):
```

Comment:
How about: `TestMakeSubmissionWhenSubmittingToChallengesUserDoesntParticipate` -> `TestMakeSubmissionWhenUserDidntParticipate`? It seems to be smaller.

Reply:
Thanks for the suggestion! I'll take that :+1:

Reply:
@vkartik97 Done!